pycytominer/operations/variance_threshold.py

```diff
@@ -25,9 +25,14 @@ def variance_threshold(
     samples : list or str, default "all"
         List of samples to perform operation on. If "all", use all samples to calculate.
     freq_cut : float, default 0.05
-        Ratio (2nd most common feature val / most common).
+        Ratio (2nd most common feature val / most common). Must range between 0 and 1.
+        Remove features lower than freq_cut. A low freq_cut will remove features
+        that have large difference between the most common feature and second most
+        common feature. (e.g. this will remove a feature: [1, 1, 1, 1, 0.01, 0.01, ...])
```

Comment:
```suggestion
        that have large difference between the most common feature value and second most
        common feature value. (e.g. this will remove a feature: [1, 1, 1, 1, 0.01, 0.01, ...])
```
Is this correct? If so, then the documentation in `pycytominer/feature_select.py` should also be changed.